### PR TITLE
Adds styling for responsiveness of the filters

### PR DESCRIPTION
**Pull request by ItsMurumba**
Clickup Task: 
1. https://app.clickup.com/t/86c0edt2y
2. https://app.clickup.com/t/86c0fd2v2

### CODE REVIEW

File: packages/transaction-log/src/components/common/animated.table.row.component.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const AnimatedTableRow = ({
   const [completed, setCompleted] = useState(false)
 
   useEffect(() => {
-    (async () => {
+    ;(async () => {
```
Comment (drizzentic) [inline packages/transaction-log/src/components/common/animated.table.row.component.tsx:16]:
```suggestion
    ;(async () => {
```
```suggestion
    (async () => {
```

File: packages/transaction-log/src/components/common/transactionlog.datatable.component.tsx [hunk 2]
```diff
@@ -153,43 +167,31 @@ const TransactionLogTable: React.FC<{
...
-                                  : transaction.status === 'Completed'
-                                  ? 'warning.main'
-                                  : transaction.status === 'Completed with error(s)'
-                                  ? 'warning.main'
-                                  : 'error.main',
-                            }}
-                          />
-                        </IconButton>
+                        />
+                      </IconButton>
```
Comment (drizzentic) [inline packages/transaction-log/src/components/common/transactionlog.datatable.component.tsx:194]: Suggest to use text and not button based on this feedback https://app.clickup.com/t/86c0fd2v2

**Review by drizzentic — CHANGES_REQUESTED** [review]
LGTM! Just some minor changes

File: packages/transaction-log/src/components/common/transactionlog.datatable.component.tsx (hunk 2 above)
Comment (ItsMurumba) [reply packages/transaction-log/src/components/common/transactionlog.datatable.component.tsx:193]: Fixed.

**Review by drizzentic — APPROVED** [review]

**Pull request merged**